### DISCUSSION TITLE
[Feature] Desktop Support

### DISCUSSION THREAD

**Discussion by kaysond** [immich-app/immich#4914] [Feature Request]
### The feature

I want to move my wife off of iCloud and onto Immich. She doesn't want to, though, because she wants offline access to all of her photos on her laptop a la Photos. I think I could convince her to let go of the offline requirement, because the thing that she really is being able to use the photos directly in other desktop apps (office, mail, etc). Having to go to the browser, save the picture, then find the picture and use it elsewhere is not the most user friendly workflow.

I know there are workarounds for this (e.g. Nextcloud et al), and that's probably what I'll end up doing, but it would be great to have a first party solution!

### Platform

- [ ] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1427

**Reply by kaysond** [reply]
Can we keep this open for discussion? Just because there was no plan to do this (as of January) doesn't mean there never will be? Or that a new developer won't want to contribute it.

@alextran1502 - any thoughts?
